app/uk/gov/hmrc/bindingtariffrulingfrontend/config/AppConfig.scala

```diff
@@ -23,16 +23,16 @@ import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 @Singleton
 class AppConfig @Inject() (val configuration: Configuration) extends ServicesConfig(configuration) {
 
-  lazy val assetsPrefix: String   = loadConfig(s"assets.url") + loadConfig(s"assets.version")
-  lazy val analyticsToken: String = loadConfig(s"google-analytics.token")
-  lazy val analyticsHost: String  = loadConfig(s"google-analytics.host")
-  lazy val reportAProblemPartialUrl: String =
-    s"$contactHost/contact/problem_reports_ajax?service=$contactFormServiceIdentifier"
-  lazy val reportAProblemNonJSUrl: String =
-    s"$contactHost/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
+  private lazy val contactHost             = configuration.getOptional[String](s"contact-frontend.host").getOrElse("")
+  private val contactFormServiceIdentifier = "BindingTariffRulings"
```

Comment:
Should that be changed since we are changing all to include ATaR instead?

Reply:
Good question. I haven't changed it (it's just been moved in the file).

I'm not sure where this ID is from, but it's consistent with the format used in the Trader Frontend:
https://github.com/hmrc/binding-tariff-trader-frontend/blob/master/app/config/FrontendAppConfig.scala#L36

Reply:
@vajarova as discussed can we raise a new ticket if this ID needs to be changed in this and/or other services?

Reply:
Discussed with Matt and we will do both changes (trader FE and Ruling) together in a separate ticket